tests/gptq/test_quantization.py

```diff
@@ -213,6 +227,42 @@ def test_exllama_max_input_length(self):
             quantized_model_from_saved.generate(**inp, num_beams=1, min_new_tokens=3, max_new_tokens=3)
 
 
+class GPTQTestExllamav2(GPTQTest):
+    desc_act = False
+    disable_exllama = True
+    disable_exllamav2 = True
+
+    def test_generate_quality(self):
+        # don't need to test
+        pass
+
+    def test_serialization(self):
+        # don't need to test
+        pass
```

Comment:
why not?

Reply:
Here, we quantize the model using the cuda-old kernel and save the model to later load it with exllamav2 for the `test_exllama_serialization` test. Since these tests will use the cuda-old kernel, we don't need to test them as we already do so in a previous test.

Reply:
And how about generate_quality?

Reply:
This is also tested in the `GPTQTest` class. The wording is confusing but `test_exllama_serialization` in `GPTQTestExllamav2` does two things: test the loading quantized weights with exllamav2 kernels + test the inference correctness.